airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/test/commands.py

```diff
@@ -72,7 +72,6 @@ async def test(pipeline_context: ClickPipelineContext):
                 include=directories_to_mount,
             ),
         )
-        .with_exec(["poetry", "config", "virtualenvs.create", "false"])
```

Comment:
@erohmensing I originally made this chance because, in a containerized environment, it is not required that poetry creates virtualenvs. This change was harmless for other test suites. There's probably a discrepancy between `pyproject.toml` file of the orchestrator and others. 
We should ideally keep my change but if I can't find the root cause quickly on the orchestrator side it's not problematic to remove it.

Reply:
Edit: I was wrong.
It's better to always create virtualenvs: https://python-poetry.org/docs/configuration/#virtualenvscreate